### DISCUSSION TITLE
@attribute for jsx

### DISCUSSION THREAD

**Discussion by marlun** [nvim-treesitter/nvim-treesitter-textobjects#356] [Q&A]
I've added the following keymaps:
```
["ia"] = "@attribute.inner",
["aa"] = "@attribute.outer",
```
I'm able to use them in html and tagged template liteals like html`` (probably because of some plugin) but I'm not able to do it tsx files. Should they in inherit from html too?

**Comment by ertucode** [comment]
For anyone coming here for a solution you can use 

```
["aj"] = "@parameter.outer",
["ij"] = "@parameter.inner",
```

**Comment by kiyoon** [comment]
#751 fixed this. You should use attribute, not parameter from now.
